spec/runner_spec.rb

```diff
@@ -112,6 +113,7 @@
                                                     triable: false,
                                                     feedback: true,
                                                     queriable: false,
+                                                    multifile: false,
```

Comment:
It would be nice to add another test that validates the `multifile: true` situation.

Reply:
I've added one. It's number 100 :tada: